br/backup-and-restore-faq.md

```diff
@@ -164,6 +164,12 @@ You can use [`filter.rules`](https://github.com/pingcap/tiflow/blob/7c3c2336f981
 
 Yes. BR backs up the [`SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) information of a table. The data of the restored table is also split into multiple Regions.
 
+## What should I do if the `entry too large, the max entry size is 6291456, the size of data is 7690800` error is reported when restoring data using BR?
```

Comment:
What should I do if the restore failure with error message the `entry too large, the max entry size is 6291456, the size of data is 7690800` using BR?

Reply:
it looks 'using BR' can removed also, since this chapter is all about BR FAQ